stdlib/tarfile.pyi

```diff
@@ -81,7 +86,8 @@ class TarFile(Iterable[TarInfo]):
     dereference: Optional[bool]
     ignore_zeros: Optional[bool]
     encoding: Optional[str]
-    errors: str
+    errors: Optional[str]
```

Comment:
I think this should still be `errors: str`. I know the class level declaration has `errors = None`, but instances will end up having `errors` as a `str` and I believe it gets passed to various functions that expect str.
```suggestion
    errors: str
```

Reply:
Removed the optional part from type hinting as mentioned